packages/discord.js/src/structures/Presence.js

```diff
@@ -285,6 +292,13 @@ class Activity {
  */
 class RichPresenceAssets {
   constructor(activity, assets) {
+    /**
+     * The activity of the RichPresenceAssets
+     * @type {Activity}
+     * @private
```

Comment:
```suggestion
```
Not sure if this should be private

Reply:
Also not sure but cool that they will be accesable to users